src/components/IssueListItem.js

```diff
@@ -1,22 +1,30 @@
 import React from "react";
 import {FloatRight, FloatLeft, FlexColumn, FlexHeader, FlexCenter} from "../styles/Grid";
 import {chevronRight} from "../icons";
+import moment from "moment";
```

Comment:
Since we are only using one function from the moment. Can we consider using dayjs instead? My consideration is for size reasons, the moment library is 250 times larger than dayjs.

https://day.js.org/docs/en/display/from-now

Reply:
@bdougie Great 😍 tomorrow I'll work on it!